src/templates/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "<%- appName %>",
+  "version": "0.0.0-pre",
+  "description": "A newly scaffolded Dojo 2 App",
+  "private": true,
```

Comment:
does it make sense for our d2 apps to have private set to `true` in the package.json